tools/diff-processor/labels/get_issue.go

```diff
@@ -0,0 +1,46 @@
+package labels
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	labeler "github.com/GoogleCloudPlatform/magic-modules/tools/issue-labeler/labeler"
+)
+
+func GetIssue(repository string, id uint64) (labeler.Issue, error) {
```

Comment:
recommend maintaining a github library similar to 
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/magician/github/init.go

If you expect more github dependencies outside of the labels sub-package

Reply:
yeah, I had been thinking it would make sense to merge this with the issue-labeler's github functions but didn't want to include a bunch of refactoring of the issue-labeler with this change. But hey, the Magician already has a library I could just add this to - what would you think about that?

Reply:
I'm not against it per say. I think there is a question of whether that would make the build slower (importing magician into this package means more dependencies for this application). I agree, centralizing it would be nice. I'll leave the call to you.

Reply:
Okay - then I'll leave it here for now and we can evaluate centralizing later. It does seem like it could be valuable.